api/openapi.yaml

```
@@ -140,11 +140,11 @@ paths:
           description: Infra ID
           required: true
         - in: query
-          name: include_geom
+          name: exclude_extensions
           schema:
             type: boolean
             default: false
-          description: Whether the railjson should contain geometry information
+            description: Whether the railjson should contain extensions
```

This is the description of the query parameter not the schema. Are you using a swagger while editing the openapi?

```suggestion
          description: Whether the railjson should contain extensions
```